namespace cpp PropertyTypes

enum Types 
{
	Exit,
	Ingestable,
	Poison,
	Light,
	Weapon,
	FireWeapon,
	Missile,
	Armor,
	Container,
	Mountable,
	Decay,
	Money,
	Stats,

	TOP					// Just to be sure.
}


